lib/hammer_cli_katello/repository.rb

```diff
@@ -220,6 +221,8 @@ def execute
           File.open(file_path, 'rb') { |file| upload_file file }
```

Comment:
The extra option to publish the repository could be given to the user or left out entirely, and this code here could be changed to upload the files much smarter with one call to `import_uploads` instead of N calls. [Your Katello code](https://github.com/Katello/katello/pull/6544) would then only publish once on the last upload rather than forcing `repository_publish: false` in hammer and then triggering a publish. It just seems like we have this logic in two places, and it's a non-ideal use of our API.

That said, this works great, and my above comment would be a performance improvement on top of the bug fix, so I would ACK this if you create an issue for cleaning up this code to only call import_upload once.